sys/mips/malta/obio.c

```diff
@@ -85,7 +85,7 @@ obio_attach(device_t dev)
 	struct obio_softc *sc = device_get_softc(dev);
 
 	sc->oba_st = mips_bus_space_generic;
-	sc->oba_addr = MIPS_PHYS_TO_KSEG1(MALTA_UART0ADR);
+	sc->oba_addr = (vm_ptr_t)MIPS_PHYS_TO_KSEG1(MALTA_UART0ADR);
```

Comment:
Still confused by all these... will stop commenting on them as I'll be here all day.